packages/wasm-edit/src/index.js

```diff
@@ -31,7 +31,7 @@ function sortBySectionOrder(nodes: Array<Node>) {
     }
 
     // $FlowIgnore ensured above
```

Comment:
Can the FlowIgnore can be removed now? Did it hide the error?

Reply:
Yes indeed, thanks.